pages/_document.js

```diff
@@ -1,40 +1,10 @@
-import Document, { Html, Head, Main, NextScript } from "next/document";
-
-const GTM_ID = process.env.GTM_ID;
+import Document, { Html, Head, Main, NextScript } from 'next/document';
```

Comment:
Why are we using `class component` instead of `function` in _document.js?

Reply:
Because it was that already, and yet again, this PR does not want to edit any code. This is for another PR targeted at Code Fixtures.